extensions/amp-date-picker/amp-date-picker.md

```diff
@@ -182,7 +182,7 @@ and the user can select a single date. This is the default selection type.
 </amp-date-picker>
 ```
 
-<!-- TODO(cvializ): add image -->
+![A static single date picker](img/amp-date-picker-single-static.png)
```

Comment:
the images are awesome addition, they seem a bit too big and take too much real state from the actual text, maybe scale them down to 50%? (unfortunately it doesn't look like Github supports `=npx` MD variant for scaling down images at runtime)